testinfra/modules/environment.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import unicode_literals
+import json
+from testinfra.modules.base import InstanceModule
+
+
+class Environment(InstanceModule):
+    """Get Environment variables
+
+    >>> host.environment()
+    {
+        "EDITOR": "vim",
+        "SHELL": "/bin/bash",
+        [...]
+    }
+    """
+
+    def __call__(self):
+        cmd = "python -c 'import json, os;print(json.dumps(dict(os.environ)))'"
+        return json.loads(self.check_output(cmd))
```

Comment:
I'd rather rely on `env` command since python may not be available on target host.

Reply:
changes have been commited to rely on env